src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,286 @@
+    true,
+    true,
+    true,
+    $location`The Secret Government Laboratory`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "FakeMedium",
+    30,
+    true,
+    "questESpFakeMedium",
+    0,
+    "",
+    true,
+    true,
+    true,
+    $location`The Secret Government Laboratory`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "Gore",
+    20,
+    true,
+    "questESpGore",
+    100,
+    "goreCollected",
+    true,
+    false,
+    false,
+    $location`The Secret Government Laboratory`,
+    $item`gore bucket`,
+    "10 meat"
+  ),
+  new conspiracyQuestData(
+    "JunglePun",
+    20,
+    false,
+    "questESpJunglePun",
+    11,
+    "junglePuns",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`encrypted micro-cassette recorder`,
+    "mys"
+  ),
+  new conspiracyQuestData(
+    "OutOfOrder",
+    30,
+    true,
+    "questESpOutOfOrder",
+    99,
+    "",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`GPS-tracking wristwatch`,
+    "10 Initiative"
+  ),
+  new conspiracyQuestData(
+    "Smokes",
+    30,
+    true,
+    "questESpSmokes",
+    10,
+    "",
+    false,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`none`,
+    "10 item, 10 pickpocket chance"
+  ),
+] as const;
+
+export function available(): boolean {
+  return have() || get("_spookyAirportToday");
+}
+
+export function completedOneTimeQuests(): boolean {
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
+    if (quest.reward === 30 && get(quest.questStateProperty) !== "finished") {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+export function haveQuest(): boolean {
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
+    if (get(quest.questStateProperty) !== "") {
+      return true;
+    }
+  }
+  return false;
+}
+
+export function activeQuest(): string {
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
+    if (
+      get(quest.questStateProperty) !== "unstarted" &&
+      get(quest.questStateProperty) !== "finished"
+    ) {
+      return quest.name;
+    }
+  }
+  return "";
+}
+
+export function getQuest(): void {
+  if (get(`_questESp`) === "" && !haveQuest() && canInteract() && available()) {
+    visitUrl(radio);
+    runChoice(1);
+  }
+}
+
+export function questComplete(): boolean {
+  if (!haveQuest()) return false;
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
+    if (
+      quest.questStateProperty ===
+      (quest.requiredItem === $item`none` ? "step1" : "step2")
+    ) {
+      return true;
+    }
+  }
+  return false;
+}
+
+export function hasActiveQuest(): boolean {
+  return haveQuest() && !questComplete();
+}
+
+export function turnInQuest(): void {
+  if (questComplete()) {
+    visitUrl(radio);
+    runChoice(1);
+  }
+}
+
+export function activateProtocol(protocol: number) {
```

Comment:
Make an enum or something for these protocols, so people can refer to them by useful names